src/components/TaxPayer/PersonFields.tsx

```diff
@@ -15,37 +16,41 @@ interface PersonFieldsProps<T extends Person> extends BaseFormProps {
   defaults?: T
   children?: ReactNode
   errors: DeepMap<Partial<Person>, FieldError>
+  person?: Person | null | undefined
   control: Control
 }
 
-export const PersonFields = <T extends Person>({ register, control, errors, defaults, children }: PersonFieldsProps<T>): ReactElement => (
+export const PersonFields = <T extends Person>({ register, control, errors, defaults, children, person }: PersonFieldsProps<T>): ReactElement => (
   <div>
     <LabeledInput
       label="First Name and Initial"
+      aria-label="First Name and Initial"
```

Comment:
Material UI components bring accessibility out of the box, but our custom elements don't always have the same attributes. I chucked in `aria-label` here, but it'd be worth raising a separate issue to do a full accessibility audit. 

Lighthouse gave us an 87, but it could be better 😄

Reply:
From this doc:

> The aria-label attribute is used to define a string that labels the current element. Use it in cases where a text label is not visible on the screen. If there is visible text labeling the element, use aria-labelledby instead. 

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

Why add aria-label?

There probably is an accessibility issue though, because the label in `<LabeledInput>` is just in a separate `p` tag floating above the input:

```
      <Box display="flex" justifyContent="flex-start">
        <p><strong>{strongLabel}</strong>{label}</p>
      </Box>
      <Box display="flex" justifyContent="flex-start">
        {input}
      </Box>
```
That's probably not right. Should be a separate issue though.